### DISCUSSION TITLE
When will Node.js 16.4.0 be published in Docker Hub?

### DISCUSSION THREAD

**Discussion by hongbo-miao** [nodejs/help#4863] [Q&A] [answered]
Node.js 16.4.0 has been published for 3 days.
I am wondering when will Node.js 16.4.0 be published in Docker Hub and is publishing work manually done currently?
If it is, just a friendly remind in case someone forget to publish. Thanks!

**Comment by richardlau** [comment, marked as answer]
You can track it https://github.com/nodejs/docker-node/pull/1502. There's currently issues verifying gpg signatures during the build process due to the sks-keyservers.net servers no longer work https://github.com/nodejs/docker-node/issues/1500.

**Reply by hongbo-miao** [reply]
I see, thanks for the info @richardlau !
